packages/blocks/shuffle/src/components/add-entities-dialog.tsx

```diff
@@ -0,0 +1,188 @@
+import { Entity, EntityType, GraphBlockHandler } from "@blockprotocol/graph/.";
+import {
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  FormControl,
+  FormGroup,
+  FormControlLabel,
+  Checkbox,
+  formControlLabelClasses,
+  experimental_sx as sx,
+  styled,
+} from "@mui/material";
+import { Button } from "@hashintel/hash-design-system";
+import { useEffect, useState } from "react";
+import { v4 as uuid } from "uuid";
+import { Item } from "../shuffle";
+import { getEntityLabel } from "../utils";
+
+const SFormControlLabel = styled(FormControlLabel)(
+  sx({
+    width: "100%",
+
+    [`.${formControlLabelClasses.label}`]: {
+      overflow: "hidden",
+      whiteSpace: "nowrap",
+      textOverflow: "ellipsis",
+    },
+  }),
+);
+
+interface AddEntitiesDialogProps {
+  entityTypes: EntityType[];
+  blockEntityId: string;
+  onAddEntityItems: (items: Item[]) => void;
+  graphService?: GraphBlockHandler | null;
+  open: boolean;
+  handleClose: () => void;
+}
+
+export const AddEntitiesDialog = ({
+  entityTypes,
+  blockEntityId,
+  onAddEntityItems,
+  graphService,
+  open,
+  handleClose,
```

Comment:
I think we agreed props of this kind should be prefixed with on - I would name this onClose

Reply:
Ooops 🤦‍♂️ , totally right, updated here https://github.com/hashintel/hash/pull/1033/commits/baf04ca1eacd1f668f128e2a2bd1c260bc05203c